rust/ohkami-nio/Cargo.toml

```diff
@@ -5,15 +5,9 @@ edition = "2021"
 publish = false
 
 [profile.release]
-opt-level = 3
-debug = false
-debug-assertions = false
-lto = true
-panic = "abort"
-incremental = false
+lto           = true
+panic         = "abort"
 codegen-units = 1
```

Comment:
not needed 😛 

compilation options are in `rust/Dockerfile`

what is codegen-units for ?